eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/contig/ContigMappingTest.java

```diff
@@ -0,0 +1,191 @@
+package uk.ac.ebi.eva.accession.dbsnp.contig;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class ContigMappingTest {
+
+    private static final String GENBANK_1 = "genbank_example_1";
+
+    private static final String GENBANK_2 = "genbank_example_2";
+
+    private static final String GENBANK_3 = "genbank_example_3";
+
+    private static final String REFSEQ_1 = "refseq_example_1";
+
+    private static final String REFSEQ_2 = "refseq_example_2";
+
+    private static final String REFSEQ_3 = "refseq_example_3";
+
+    private static final String SEQNAME_1 = "1";
+
+    private static final String SEQNAME_2 = "2";
+
+    private static final String SEQNAME_3 = "3";
+
+    private static final String SEQNAME_ch1 = "ch1";
+
+    private static final String UCSC_1 = "ucsc_example_1";
+
+    private static final String UCSC_2 = "ucsc_example_2";
+
+    private static final String UCSC_3 = "ucsc_example_3";
+
+    private static final String GENBANK_CONTIG = "CM000994.2";
+
+    private static final String REFSEQ_CONTIG = "NC_000067.6";
+
+    private static final String SEQNAME_CONTIG = "1";
+
+    private static final String UCSC_CONTIG = "1";
+
+    private static final String CONTIG_WITHOUT_SYNONYM = "NT_without_synonym";
+
+    private ContigMapping contigMapping;
+
+    @Before
+    public void setUp() throws Exception {
+        String fileString = Thread.currentThread().getContextClassLoader().getResource("input-files/AssemblyReport.txt")
+                                  .toString();
```

Comment:
It is usually preferable to use the current class loader by calling `getClass().getClassLoader().getResource("/input-files/AssemblyReport.txt").toString()`. Note the forward slash at the beginning of the path.